recipe/meta.yaml

```diff
@@ -36,23 +39,29 @@ requirements:
     - pillow >=3.1.1
     - pyct >=0.4.5
     - scipy
-    - toolz >=0.7.4
     - xarray >=0.9.6
+  run_constrained:
+    # 2022/7/22: ImportError: Numba needs NumPy 1.21 or less
```

Comment:
@skupr-anaconda - What is the relationship between `datashader` and `numba`? It's not clear to the reader why `numba`'s pinning of `numpy` is important.

Reply:
Oh wait I see: https://github.com/holoviz/datashader/blob/v0.14.1/setup.py#L17